features/ncs/ncs-northbounder/src/main/java/org/opennms/netmgt/ncs/northbounder/NCSNorthbounderConfig.java

```diff
@@ -115,10 +115,12 @@ HttpEntityEnclosingRequestBase getRequestMethod(URI uri) {
     @XmlAttribute(name="fragment", required=false)
     private String m_fragment;
 
+    @XmlAttribute(name="use-system-proxy")
```

Comment:
I didn't find any documentation for this configuration. @agalue did I miss it or is there none?